docs/Using Fleet/enroll-chromebooks.md

```diff
@@ -24,6 +24,22 @@ By default, the hostname for a Chromebook host will be blank. The hostname can b
 ## Debugging ChromeOS
 To learn how to debug the Fleetd Chrome extension, visit [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Testing-and-local-development.md#fleetd-chrome-extension).
 
+## Potential issues and trubleshooting: 
+- 1 The extension does not install on our ChromeBooks.
+- 2 Chrome Web Browsers on other OSs (Mac/Linux/Windows) get this extension (where it's not needed).
+
+Google Admin is arranged in a hirarchy of Organizational Units (OUs) tree. Each of the OUs can hold a combination of USERs and/or DEVICEs.
+Chrome extensions can be set for a specific OU (force-installed, allow install or block). However, Extensions can only be set at USERs level (not DEVICES).
+If a chrome extension is deployed to an OU that only has DEVICES, it will not be installed. 
+On the other hand if you deploy an extension to an OU that hold USERS with both ChromeBooks and
+managed Chrome web browsers (e.g. Chrome browser on a MacBook), it will deploy the extension to that Chrome Web Browser.
+
+### Our recommendation: 
+- Create an OU that will hold all USERs with ChromeBooks. Deploy our extension to it (Force-Install).
```

Comment:
*Chromebooks

Replace the word it with "the OU" and a link to our instructions how to deploy

Reply:
Update the rest of the doc with consistent capitalization.